project2/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+
+
+
+
+
+
+<body>
+
+
+
+
+
+    <div class="card">
+        <img src="Healthy-Garlic-Shrimp-Pasta-with-Lemon.jpg" alt="">
+
+        <h1>Garlic Shrimp Pasta
+        </h1>
+
+
+        <ul class="menu">
+            <li><a href="#fas fa-shopping-cart"><i class="fas fa-shopping-cart"></i></a></li>
+            <li><a href="#fas fa-utensil-spoon"><i class="fas fa-utensil-spoon"></i></a></li>
+            <li><a href="#fas fa-clock"><i class="fas fa-clock"></i></a></li>
+        </ul>
+
+
+
+        <section id="fas fa-shopping-cart">
+            <p>
+                <h2><b>Ingredients</b></h2>
+                <ul>
+                    <li>8 ounces whole-wheat linguine</li>
+                    <li>1 pound fresh or frozen raw shrimp</li>
+                    <li>2 tablespoons extra-virgin olive oil</li>
+                    <li>1 shallot</li>
+                    <li>5 cloves garlic</li>
+                    <li>1/2-1 teaspoon crushed red pepper flakes</li>
+                    <li>1 teaspoon kosher salt</li>
+                    <li>1 teaspoon black pepper</li>
+                    <li>24 ounces mixed vegetables</li>
+                    <li>1/4 cup dry white wine</li>
+                    <li>Zest and juice of 1 large or 2 small lemons</li>
+                    <li>3 tablespoons freshly grated Parmesan cheese</li>
+                    <li>2 tablespoons chopped fresh parsley</li>
+
+                </ul>
+            </p>
+        </section>
+
+        <section id="fas fa-utensil-spoon">
+            <h2><b>Instructions</b></h2>
+
+            <ol>
+                <li>Bring a large pot of salted water to a boil and cook the pasta until al dente, according to
+                    package directions. Reserve 1/4 cup pasta water, drain remaining water, and then toss
+                    with a bit of olive oil to prevent sticking. Set aside.</li><br>
```

Comment:
You should not use the `br` element to add spacing in this way -- instead, you want to increase the `margin-bottom` of the `li` element in your CSS.